### PR TITLE
Updated Change Log with Info About Recent Merged Pull Requests

### PR DESCRIPTION
**Pull request by liangfok**
We should keep this document updated as much as possible.

Related: https://github.com/RobotLocomotion/drake/issues/1882

### CODE REVIEW

**Comment by jwnimmer-tri** [comment]
This is going too far.  The purpose of a changelog is to _summarize_ the _relevant_ changes to downstream users.   This level of detail is too much for downstream to care about.

**Comment by liangfok** [comment]
Should I remove all PRs that do not result in API changes?

**Comment by jwnimmer-tri** [comment]
Yes.  Also, instead of re-summarizing the details of the PR in the supplemental text, just give a good title and let the user click through to read about the details.

This is also especially hard without a release process.  Ideally, we would sort/group these items to help downstream grok them, but for now a livestream is about the best we can do.

**Comment by liangfok** [comment]
Related: https://github.com/olivierlacan/keep-a-changelog

**Comment by liangfok** [comment]
@jwnimmer-tri: Is there a `grok` tool I should know about?

**Comment by jwnimmer-tri** [comment]
Nope.  See [jargon file entry](http://www.catb.org/jargon/html/G/grok.html).

**Comment by liangfok** [comment]
PTAL. I've shortened the change log to only include PRs that result in API or command line option changes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,20 @@ Drake is undergoing continuous development and cannot yet offer a stable, docume
 Descriptions of Pull Requests
 ------------------------------
 
-### ([@liangfok][], [#1953][]) - Replacement of valuecheckMatrix() with CompareMatrices()
+
+### ([@liangfok][], [#1985][]) - Added Max Delay and Warn command line options to quadrotor's unit test
```
Comment (jwnimmer-tri) [inline CHANGELOG.md]: 1985 need not appear.  The defaults are the same, and its unlikely to be anything users care to play with in terms of a new feature.

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,4 +35,25 @@ should be modified to use Google Test and the following line:
...
+[#1986]: https://github.com/RobotLocomotion/drake/issues/1986
+[#1988]: https://github.com/RobotLocomotion/drake/issues/1988
+[#1990]: https://github.com/RobotLocomotion/drake/issues/1990
+[@billhoffman]: https://github.com/billhoffman
+[@chrisdembia]: https://github.com/chrisdembia
+[@david-german-tri]: https://github.com/david-german-tri
+[@jamiesnape]: https://github.com/jamiesnape
+[@jwnimmer-tri]: https://github.com/jwnimmer-tri
+[@liangfok]: https://github.com/liangfok
+[@patmarion]: https://github.com/patmarion
```
Comment (jwnimmer-tri) [inline CHANGELOG.md]: Most of these are deadwood now, aren't they?

**Comment by liangfok** [comment]
@jwnimmer-tri: Thanks for the code review. PTAL.

**Comment by jwnimmer-tri** [comment]
I added a few notes, but then gave up, because bashing this into shape by saying what not to do over many iterations of PR review of patches is inefficient.  I will be happy to pair program this with you next time we are co-located, or else grab David, or I can write up my ideal patch, or something.

**Comment by liangfok** [comment]
OK I will go ahead and close this PR until we can work this out F2F.

**Pull request closed without merging**